db/seeds.rb

```diff
@@ -7,23 +7,23 @@
 #   Character.create(name: "Luke", movie: movies.first)
 
 
-first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
-second_user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
-
-first_post = Post.create(author: first_user, title: 'Bonjour', text: 'Greeting in french')
-second_post = Post.create(author: first_user, title: 'Guess Game', text: 'Dont guess all the time, but do so when you are sure')
-third_post = Post.create(author: first_user, title: 'Food is sweet', text: 'Bread is universally eaten')
-fourth_post = Post.create(author: first_user, title: 'Holla', text: 'Spanish greeting')
-fifth_post = Post.create(author: second_user, title: 'Merci', text: 'Thank you microverse')
-
-comment1 = Comment.create(post: Post.first,author: User.first, text: 'Hi Lilly!' )
-comment2 = Comment.create(post: Post.first,author: User.second, text: 'Hi Tom!' )
-comment3 = Comment.create(post: Post.second,author: User.second, text: 'Hi!' )
-comment4 = Comment.create(post: Post.second,author: User.first, text: 'Hey!' )
-comment5 = Comment.create(post: Post.third,author: User.first, text: 'Hi there!' )
-comment6 = Comment.create(post: Post.third,author: User.second, text: 'Hello!' )
-
-like1 = Like.create(post: Post.first,author: User.first)
+first_user = User.create!(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+second_user = User.create!(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
```

Comment:
## [OPTIONAL]

- [ ] KIndly make sure you initialize the posts_counter to 0 when creating a new user, this will remove the error I am getting when running your seeds

<img width="649" alt="image" src="https://user-images.githubusercontent.com/65068771/231943483-6c919148-745f-4110-a9bc-d1e6cce42c41.png">